corehq/apps/reports/standard/deployments.py

```diff
@@ -139,6 +141,22 @@ def rows(self):
             )
         return rows
 
+    @property
+    def export_table(self):
+        def _fmt_ordinal(ordinal):
+            if ordinal >= 0:
+                return safe_strftime(date.fromordinal(ordinal), USER_DATE_FORMAT)
+            return 'Never'
```

Comment:
should be translated. also, did anyone weigh in on this value? my instincts would have been either an empty string or `"--"` (or whatever the couchexport default empty is)